scripts/jenkins-yoctobuild-build.sh

```diff
@@ -286,13 +276,6 @@ EOF
         cat >> $BUILDDIR/conf/local.conf <<EOF
 PREFERRED_VERSION_pn-mender-artifact = "$mender_artifact_version-git%"
 PREFERRED_VERSION_pn-mender-artifact-native = "$mender_artifact_version-git%"
-EOF
-    fi
-
-    if is_poky_branch morty; then
-        # Morty needs oe-meta-go
-        cat >> $BUILDDIR/conf/bblayers.conf <<EOF
-BBLAYERS_append = " $WORKSPACE/oe-meta-go"
```

Comment:
You should also remove `oe-meta-go` from the `git clone` section.

Reply:
Thanks! Removed.